usaspending_api/etl/swap_in_new_table_strategies.py

```diff
@@ -0,0 +1,944 @@
+from abc import ABC, abstractmethod
+from argparse import ArgumentError
+from datetime import datetime
+import json
+from pprint import pformat
+import re
+from django.db import ProgrammingError, connection, transaction
+import logging
+from usaspending_api.broker.helpers.last_delta_table_load_version import (
+    get_last_delta_table_load_versions,
+    update_last_live_load_version,
+)
+from usaspending_api.broker.lookups import LoadTrackerLoadTypeEnum, LoadTrackerStepEnum
+from usaspending_api.common.helpers.sql_helpers import (
+    get_parent_partitioned_table,
+    is_table_partitioned,
+    ordered_dictionary_fetcher,
+)
+from usaspending_api.common.load_tracker import LoadTracker
+
+from django.core.management import BaseCommand
+from typing import List
+from usaspending_api.etl.management.helpers.table_specifications import DATABRICKS_GENERATED_TABLE_SPEC as TABLE_SPEC
+from usaspending_api.search.delta_models.award_search import AWARD_SEARCH_COLUMNS
+from usaspending_api.etl.management.sql.swap_in_new_table.update_live_tables import update_live_tables_sql
+from usaspending_api.etl.management.sql.swap_in_new_table.insert_to_live_tables import insert_to_live_tables_sql
+from usaspending_api.etl.management.sql.swap_in_new_table.delete_from_live_tables import delete_from_live_tables_sql
+from usaspending_api.etl.management.sql.swap_in_new_table.dependent_views import detect_dep_view_sql
+
+COLUMN_SPECS = {"award_search": AWARD_SEARCH_COLUMNS}
```

Comment:
addressed

Reply:
This is still referencing a new set of data about the table `TABLE_COLUMN_SPECS` and not the already existing `TABLE_SPEC` definition.  I believe everything needed for this class should already be there. `AWARD_SEARCH_COLUMNS` and `unique_identifiers` should cover the needs with some tweaking.

Reply:
It's an entirely new data structure that stores new data not available. I'm confused why this is a problem?

Reply:
I would have preferred to keep this configuration all in one place. I believe `unique_identifiers` would have provided enough info to replace `delete_column`, `null_column`, and `join_condition`. This would make it easier to perform the tech debt of refactoring the TABLE_SPEC info into the Django model files. 

I'm not going to let this hold up the PR review though.